samples/dashboards/demos/climate/demo.js

```diff
@@ -36,7 +36,7 @@ async function setupDashboard() {
     citiesData = await buildCitiesData();
     buildSymbols();
 
-    const cityStore = await dataPool.getStore(cityScope);
+    const cityConnector = await dataPool.getConnector(cityScope);
```

Comment:
That is a question that can be raised with the previous name `cityStore` as well. I would like to have the variable names as short as possible because of the 80 character limit.